Update peerDependencies to use laravel-elixir@5.0

``` bash
npm ERR! peerinvalid The package laravel-elixir@5.0.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer laravel-elixir-livereload@1.1.4 wants laravel-elixir@^4.0
```

--- a/package.json
+++ b/package.json
@@ -31,7 +31,7 @@
     "gulp-util": "~2.2.0"
   },
   "peerDependencies": {
-    "laravel-elixir": "^4.0"
+    "laravel-elixir": "^5.0"
   },
   "engines": {
     "node": ">=0.8.0",

